qiskit/algorithms/time_evolvers/pvqd/pvqd.py

```diff
@@ -369,7 +369,7 @@ def evolve(self, evolution_problem: TimeEvolutionProblem) -> TimeEvolutionResult
             )
             observable_values = [evaluate_observables(self.initial_parameters)]
 
-        fidelities = [1]
+        fidelities = [1]  # TODO: should be float?
```

Comment:
Since Python does float calculations per default, it doesn't matter if we write 1 or 1.0 from a functionality perspective. You could also change to `[1.0]`, personally I don't have a strong preference